lib/crypto/bn.js

```diff
@@ -1,9 +1,78 @@
 'use strict';
 
-var BN = require('bn.js');
 var $ = require('../util/preconditions');
 var _ = require('lodash');
 
+class BN {
+  #value
+
+  constructor(number, base, endian) {
```

Comment:
Do we have this wrapper somewhere else? Maybe in currency? To unify I mean

Reply:
no
a fully compatible bn.js replacement is not this clean
here, bn.js usage is very limited so it can be replaced

also, this was still using bn.js@2